src/Subscriptions/LegacyListeners/DispatchGiveSubscriptionPostCreate.php

```diff
@@ -34,6 +35,7 @@ public function __invoke(Subscription $subscription)
             'profile_id' => $subscription->gatewaySubscriptionId,
         ];
 
+        Hooks::doAction('give_subscription_inserted', $subscription->id, $args);
```

Comment:
@JasonTheAdams, I thought we could trigger this action hook from the existing dispatch handler. I am open to creating another dispatch class in case you disagree.

Reply:
@ravinderk Interesting, so these two hooks do effectively the same thing?

Reply:
@JasonTheAdams Yes. Let me know if you have any questions